src/github.com/getlantern/flashlight/client/client.go

```diff
@@ -179,3 +191,29 @@ func (client *Client) Configure(cfg *ClientConfig, proxyAll func() bool) {
 func (client *Client) Stop() error {
 	return client.l.Close()
 }
+
+func (client *Client) proxiedDialer(orig func(network, addr string) (net.Conn, error)) func(network, addr string) (net.Conn, error) {
```

Comment:
I consolidated the dialing logic into on place that's used by both intercept() and ReverseProxy.